src/pages/Error/Error500.tsx

```diff
@@ -71,7 +71,7 @@ export function Error500(): ReactElement {
             We have encountered an error. Visit the platform homepage for
             additional resources or contact our support staff.
           </span>
-          <LinkVisitHomepage />
+          <LinkVisitHomepage isRouterLink={false} />
```

Comment:
I was unable to programmatically clear the `auth.error` object, which caused endless redirecting in `App.tsx` above, so this forces a page reload and clears the auth error.